analyzers/src/SonarAnalyzer.Common/AnalysisContext/SonarAnalysisContextBase.cs

```diff
@@ -83,7 +83,7 @@ public ProjectConfigReader ProjectConfiguration()
     /// <summary>
     /// Reads the properties from the SonarLint.xml file and caches the result for the scope of this analysis.
     /// </summary>
-    public SonarLintXmlReader SonarLintFile()
+    public SonarLintXmlReader ReadSonarLintXml()
```

Comment:
It doesn't always read. And it would not be aligned with the method above.
```suggestion
    public SonarLintXmlReader SonarLintXml()
```